.github/workflows/tests.yml

```diff
@@ -203,6 +208,10 @@ jobs:
             track: "5.0/candidate"
           - test: efi-vars-editor-vm
             track: "5.0/edge"
+          - test: network-ovn ovn:latest/edge
```

Comment:
these last 2 variants stood out to me as different to the rest as they dont include a deb variant?

Reply:
The review UI doesn't make it easy to spot with our overly long exclusion matrix but yeah, that's an exclusion. So with 5.0, we test only the `ovn:deb` variant hence the exclusion of the `ovn:latest/edge` one/